pkg/debug/spec.sh

```diff
@@ -189,7 +189,7 @@ LSPCI_D=$(lspci -D)
 cat <<__EOT__
 {
   "arch": $ARCH,
-  "productURL": "$(cat /persist/status/hardwaremodel || cat /config/hardwaremodel)",
+  "productURL": "$(cat /persist/status/hardwaremodel 2> /dev/null || cat /config/hardwaremodel 2> /dev/null || echo 'not found' )",
```

Comment:
I suspect both the old and new code can product invalid URLs - in particular spaces need to be escaped in URLs.

Instead of tweaking this, isn't it better to just drop the productURL from the output in all cases?